homeassistant/components/husqvarna_automower/number.py

```diff
@@ -0,0 +1,86 @@
+"""Creates the number entities for the mower."""
+
+from collections.abc import Callable
+from dataclasses import dataclass
+import logging
+
+from aioautomower.exceptions import ApiException
+from aioautomower.model import MowerAttributes
+
+from homeassistant.components.number import NumberEntity, NumberEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import AutomowerDataUpdateCoordinator
+from .entity import AutomowerBaseEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+@dataclass(frozen=True, kw_only=True)
+class AutomowerNumberEntityDescription(NumberEntityDescription):
+    """Describes Automower number entity."""
+
+    exists_fn: Callable[[MowerAttributes], bool] = lambda _: True
```

Comment:
Why do we add this if we only have 1 entity and its true every time?

Reply:
I've just added the explanation to the PR description. Some mowers return here just `None`, so these get not added.

Reply:
But in this PR they do not return None :P